app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
+    if len(command_list) != 3:
+        raise ValueError(f"Command must have 3 elements, but got: "
+                         f"{len(command_list)}")
+    move_command, old_file_path, new_file_path = command_list
+    if move_command != "mv":
+        raise NameError(f"{move_command} is not a valid")
+    if new_file_path.count("/") == 0:
```

Comment:
Use os.path.join to concatenate parts of the path. In that case, your app will be cross-platform and will use either / or \ depending on OS